plugins/flytekit-spark/tests/test_wf.py

```diff
@@ -53,27 +58,6 @@ def my_wf() -> my_schema:
     assert df2 is not None
 
 
-def test_ddwf1_with_spark():
```

Comment:
was this test bad for some reason?

Reply:
We have the same test here. https://github.com/flyteorg/flytekit/blob/eb1c75e4fe6875640560a1d6dbdfb377c6c8e494/plugins/flytekit-spark/tests/test_wf.py#L15-L33****